prepare_cyrus.hs

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env stack
+-- stack runghc --resolver lts-8.14 --package turtle --install-ghc
```

Comment:
```
@btellier so seriously you want to keep this commit ?  https://github.com/linagora/gatling-imap/pull/22#discussion_r117443038
```

You're proposing automation for something I don't know how to do.

If you want to drop it, I'm fine with it.

If you decide to keep it, I should at least know how to use it.